app/models/review.rb

```diff
@@ -0,0 +1,9 @@
+class Review < ApplicationRecord
+  belongs_to :art, dependent: :destroy
+  belongs_to :user
+  belongs_to :booking
```

Comment:
This needs to be removed.
Instead we'll use a private method to validate that the user has a booking that is related to the art of this review.